tests/unit/backend/recipe/algorithm/test_LoadGroupingDefinition.py

```diff
@@ -311,9 +312,13 @@ def test_load_from_xml_file_with_instrument_file(self):
     #     self.do_test_load_with_instrument_name("xml")
 
     # test hdf
+    # TODO THIS IS BAD -- EWM 5043
+    @pytest.mark.xfail(strict=True)
```

Comment:
Please add of `reason="To be fixed in EWM5043"` to the call to make it more obvious